docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+|File (file)		| Use local files for reusable media.			||
+|Image (image)		| Use local images for reusable media.			| This is Drupal's built-in Image media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+|Video (video)		| A locally hosted video file.				| This is Drupal's built-in Video media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
+
+### Vocabularies (i.e. Taxonomy bundles)
+
+|Name			|Description		|Comments			|
+|---			|---			|---				|
+|Islandora Access (islandora_access)	|Terms used to limit, restrict or coordinate access	| Not sure if they actually do affect access.	|
+|Islandora Display (islandora_display)	|Terms used to alter how a repository item is viewed. ie. basic image vs large image	| Check if/how this gets populated.|
+
+### Form Modes
+
+The following "Default" forms are configured. It is unclear how/if these include Islandora-specific choices such as ordering/grouping or form widget configuration.
+
+|Type			|Bundle			|Name	|Comment|
+|---			|---			|---	|---	|
+|Node			|Islandora Object	|Default||
+|Media			|Audio			|Default||
+|Media			|Document		|Default||
+|Media			|File			|Default||
+|Media			|Image			|Default||
+|Media			|Video			|Default||
+|Taxonomy Term		|Islandora Display	|Default||
+|Taxonomy Term		|Islandora Media Use	|Default|This vocabulary is not defined in this module.|
+|Taxonomy Term		|Islandora Models	|Default|This vocabulary is not defined in this module.|
+
+
+### Display Modes
+
+The following new display modes are defined:
+
+| Type    | View Mode      | Comment                                                              |
+|---------|----------------|----------------------------------------------------------------------|
+| Content | Binary         |                                                                      |
+| Content | Open Seadragon |                                                                      |
+| Content | PDFjs          |                                                                      |
+| Content | Teaser         | This seems to be a re-definition of the built-in "Teaser" view mode. |
+| Media   | Open Seadragon |                                                                      |
+| Media   | PDFjs          |                                                                      |
+
+
+The following display modes are configured. It is unclear how/if the "Default", "Source", and "Teaser" display modes include any Islandora-specific choices such as ordering/grouping or display widget configuration. "Media EVAs (display_media)" is a view provided by the Islandora core feature.
```

Comment:
The 'source' display modes present _just_ the file.  Probably too much to throw at the user now, but that display mode is what's used by the EVA's to show the file. 

"Default" is usually our best guess as to what you want to see.  Which at this point is just basically everything with little to no formatting.